chimp.md

```diff
@@ -0,0 +1,213 @@
+| **Expected Duration** | 1 months | Start: 2nd month, **Budget Allocation: $10,500 (35%)** |
+
+---
+
+### **Milestone 3: Encrypted Launchpad & User Growth**
+
+| **Number**       | **Deliverable**                                    | **Details**                                                                                                                                                                            |
+|------------------|----------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **0a.**          | License                                            | Apache 2.0                                                                                                                                                                              |
+| **0b.**          | Documentation                                      | We will provide documentation for the **Encrypted Launchpad** module, detailing how private token sales and fundraising can be conducted on the platform.                               |
+| **0c.**          | Testing and Testing Guide                          | Unit tests will cover private fundraising, encrypted KYC (optional), and token sale management. A **testing guide** will explain how to test encrypted token sales on Chimp DEX.        |
+| **0d.**          | Docker                                             | We will provide a **Dockerfile** to test the encrypted launchpad module, enabling developers to deploy and test private fundraising pools locally.                                      |
+| **0e.**          | Article/Workshop                                   | We will publish an article on **private fundraising** through an encrypted launchpad, showcasing how privacy-oriented projects can raise funds anonymously using Chimp DEX.              |
+| **1.**           | Substrate module: Encrypted Launchpad              | We will create a Substrate module that allows projects to conduct **private token sales** and **fundraising** through encrypted KYC and token sale pools.                               |
+| **2.**           | Substrate chain                                    | This module will be integrated into the custom Substrate chain to allow **private fundraising**, making Chimp DEX the go-to platform for privacy-oriented DeFi fundraising efforts.      |
+| **3.**           | User Growth & Cross-Chain Expansion                | Launch targeted campaigns to onboard **50,000 users** and extend cross-chain compatibility to additional blockchains like **Ethereum** and **Polkadot**.                                 |
```

Comment:
What do you mean by extending to additional blockchains like Polkadot? I thought this whole product will follow a Polkadot first approach.